assets/js/create-github-issue.js

```diff
@@ -71,18 +79,22 @@ const createGithubIssue = async () => {
           user: user,
           review: review,
           reference: reference,
+          difficulty,
```

Comment:
`difficulty: difficulty`?

Reply:
It's [shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#syntax) so it evaluates to the same thing, but can change for consistency with the other fields!